packages/editor-ui/src/components/SettingsSidebar.vue

```diff
@@ -25,6 +25,17 @@
 				</i>
 				<span slot="title">{{ $locale.baseText('settings.n8napi') }}</span>
 			</n8n-menu-item>
+			<n8n-menu-item
+				v-for="fakeDoor in getFakeDoorFeatures"
+				v-bind:key="fakeDoor.featureName"
+				:index="`/settings/coming-soon?featureName=${fakeDoor.featureName}&infoText=${fakeDoor.infoText}&actionBoxTitle=${fakeDoor.actionBoxTitle}&actionBoxDescription=${fakeDoor.actionBoxDescription}&linkURL=${fakeDoor.linkURL}`"
```

Comment:
this introduces risk of XSS... lmk if it's not clear how this could be abused... just use the index as id and load data directly from store.

Reply:
Agree. Will update.